crates/core/tedge_mapper/src/c8y/mapper.rs

```diff
@@ -185,14 +224,14 @@ impl TEdgeComponent for CumulocityMapper {
                 )
                 .await?;
         }
-        let mut jwt_actor =
-            C8YJwtRetriever::builder(mqtt_config.clone(), c8y_config.bridge.topic_prefix.clone());
+
+        let mut header_actor = C8YHeaderRetriever::try_builder(&tedge_config, c8y_profile)?;
```

Comment:
```suggestion
        let mut auth_actor = C8YHeaderRetriever::try_builder(&tedge_config, c8y_profile)?;
```

Reply:
Actually `header_actor` is more accurate. It is returning only authorization header as of now, but in the future, one can modify it to return any kinds of header(s).